### DISCUSSION TITLE
Unable to build Quartznet 3.0.0 from source

### DISCUSSION THREAD

**Discussion by subhasisrout** [quartznet/quartznet#570] [General] [answered]
### running nant gave the following error - 
error MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format,please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format,please convert it to MSBuild 2003 format.

**Comment by lahma** [comment, marked as answer]
The guidance wasn't too clear on this. I've updated build instructions to be part of front page readme: https://github.com/quartznet/quartznet/blob/master/README.md . So basically you need to install Visual Studio 2017 and you should be good to go, the community version is free.

Thanks for bringing this to attention, hope this helps.
